std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
I agree about the range primitives. Personally, I think that array range primitves (except for strings) should be in object.d